lib/ssl/src/ssl.erl

```diff
@@ -2683,31 +2707,52 @@ signature_algs(Description, Version) when Description == default;
 signature_algs(Description, Version) ->
     erlang:error(badarg, [Description, Version]).
 
+
+%%--------------------------------------------------------------------
+-doc """
+Returns a list of supported elliptic curves pre TLS-1.3
+""".
+-doc(#{since => <<"OTP 27">>}).
+-spec ec_curves(Description, Version) -> NamedCurves when
+      Description :: all | default,
+      Version :: 'tlsv1.2' | 'tlsv1.1'| 'tlsv1' | 'dtlsv1.2' | 'dtlsv1',
+      NamedCurves :: [named_curve()].
+%%--------------------------------------------------------------------
+ec_curves(Description, 'dtlsv1') ->
+    ec_curves(Description, 'tlsv1.1');
+ec_curves(Description, 'dtlsv1.2') ->
+    ec_curves(Description, 'tlsv1.2');
+ec_curves(Description, Version) when Description == all;
+                                     Description == default;
+                                     Version == 'tlsv1.2';
+                                     Version == 'tlsv1.1';
+                                     Version == tlsv1 ->
+    tls_v1:ec_curves(Description, Version).
+
 %%--------------------------------------------------------------------
--doc(#{equiv => eccs/1}).
+-doc """
+Ues ec_curves/2 instead. Equivalent to
```

Comment:
typo: Ues -> Use (here and in other places)

Reply:
fixed